lib/mount.js

```diff
@@ -22,11 +22,21 @@ const mount = (app, { errLogger, logger }={}) =>
       .then(log(logger, { req, res }))
 
 const write = (req, res) => ({ body, headers={}, statusCode=200 }) => {
-  res.statusCode = statusCode
-
   for (var name in headers)
     res.setHeader(name, headers[name])
 
+  const requestEtag = req.headers['if-none-match']
+  const bodyEtag = !(body instanceof Stream) ? etag(body || '') : undefined
```

Comment:
Gotcha. Fixed in https://github.com/articulate/paperplane/pull/17/commits/ad2e958235d75a1a684218e46d426e670207f2ca